src/components/KenticoVersionsSelector.vue

```diff
@@ -23,9 +23,9 @@ import store, {
   updateSelectedCategoriesMutation,
   updateFilterSearchPhraseMutation,
   updateSelectedKenticoVersionMutation
-} from "@/store";
-import { KENTICO_VERSION_ALL_VERSIONS } from "@/utils/kenticoVersions";
-import performItemsFiltering from "@/utils/filter";
+} from "./../store";
```

Comment:
Why to use relative path ant not continue using [name module mapper](https://github.com/Kentico/ems-extension-marketplace/blob/master/package.json#L88) ?